### DISCUSSION TITLE
Swi-prolog does not provide JPL

### DISCUSSION THREAD

**Discussion by rvjansen** [Homebrew/discussions#2134] [Casks]
There is an issue **Missing JPL library in swi-prolog #32364** - and It is closed, resolved and locked. But: there is still no libjpl.dylib on macOS. Providing it seems a parameter like --with-libjpl on the build. The standard swipl is delivered as a relocatable app, but for all purposes it would be much better to have it built non-relocatable like brew does. I thought I'd ask before opening a new issue on this. I am on macOS Big Sur on an M1.

**Comment by carlocab** [comment]
As noted in Homebrew/homebrew-core#32364, using the `--with-libjpl` flag seems to make things worse.

In any case, opening another issue is likely to result in the same outcome as Homebrew/homebrew-core#32364. Please open a pull request to fix this instead.
